test/verify/machineslib.py

```diff
@@ -37,6 +37,29 @@ def readFile(name):
     return content
 
 
+# Preparations for physical disk storage pool
+def prepareDiskPool(m):
```

Comment:
This just creates the disk, it doesn't prepare any pool. So I would move inside this function also the pool preparation commands and pass as parameter the pool name.

Reply:
You mean pulling virsh commands define/create-as/start into this function ? That would make the usage of function unusable in pkg/machines/libvirtUtils.js , where its calling is followed by calling UI StoragePoolCreateDialog.
This function doesn't prepare pool, rather it's meant to "prepare enviroment for pool creation". So perhaps let's change the name of the function to "diskPoolPreparations" ?

Reply:
I guess you mean test/verify/chech-machines-dbus. But sure, I get what you mean :) In this case it's really small, 3 lines of code don't really require to be pulled into a funciton.
But if you wish to leave it like this please rename it
prepareSCSIDisk could be the new name for example.

Reply:
A renamed it into the prepareDisk and prepareSCSIDisk. Also reverted to reloading page.